docs/csharp/programming-guide/concepts/async/snippets/index/AsyncBreakfast-V2/Program.cs

```diff
@@ -10,50 +10,62 @@ static async Task Main(string[] args)
         {
             Coffee cup = PourCoffee();
             Console.WriteLine("coffee is ready");
+
             Egg eggs = await FryEggsAsync(2);
             Console.WriteLine("eggs are ready");
+
             Bacon bacon = await FryBaconAsync(3);
             Console.WriteLine("bacon is ready");
+
             Toast toast = await ToastBreadAsync(2);
             ApplyButter(toast);
             ApplyJam(toast);
             Console.WriteLine("toast is ready");
+
             Juice oj = PourOJ();
             Console.WriteLine("oj is ready");
-
             Console.WriteLine("Breakfast is ready!");
         }
+        // </SnippetMain>
```

Comment:
I moved the SnippetMain in a previous PR to include the other methods, as their implementations wasn't in the article.
Now, that this is moved back, are they included somewhere else in the article?
I don't have access to the review site to check

Reply:
This link doesn't work for you?
https://review.docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/async/?branch=pr-en-us-18555

Reply:
Yes, the review site works only for employees.

I noticed now that these versions are fully displayed, so it's good it's moved back to its original place.